### ISSUE TITLE
Attaching ref description in readme is incorrect

### ISSUE THREAD

**Issue by simonDos** [moaazsidat/react-native-qrcode-scanner#40]
<QRCodeScanner ref={(node) => {this.scanner = node}}>

instead of

<QRCodeScanner ref={(node) = this.scanner = node}>

solved it for me at least. PS thanks for the module :)

**Comment by moaazsidat** [comment]
yep, you are correct, I've updated the readme and will push it out in the next release, thanks for the catch!
